onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -12,6 +12,18 @@
 namespace onnxruntime {
 namespace perftest {
 
+#ifdef _MSC_VER
+// Convert a wide Unicode string to an UTF8 string
+std::string utf8_encode(const std::wstring &wstr)
```

Comment:
remove and use https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/common/str_helper.cc#L9  ?

Reply:
@jywu-msft  Done. Even verified the new changes locally on linux and windows builds as well. It is working fine.